other-docs/guides/upgrading/v5.md

```diff
@@ -0,0 +1,143 @@
+
+
+### WordPress 5.5 "Eckstine"
+
+The latest CMS module includes WordPress 5.5. Some of the highlights of this release are:
+
+* [Block patterns](https://make.wordpress.org/core/2020/07/16/block-patterns-in-wordpress-5-5/) allow users to build their own layouts and presets
+* [Inline image editing](https://make.wordpress.org/core/2020/07/20/editing-images-in-the-block-editor/) in the block editor
+
+You can [find the full WordPress 5.5 release notes here](https://wordpress.org/news/2020/08/eckstine/), and the developer [field guide here](https://make.wordpress.org/core/2020/07/30/wordpress-5-5-field-guide/). Some internal APIs and compatibility may have changed, especially in block editor APIs, so ensure you consult the field guide for a full list of changes.
+
+Note that in line with our development and user experience philosophies, some changes in the CMS module may differ from WordPress changes. Notably many of the major features in WordPress 5.5 are in the autoupdate system, which is disabled in Altis.
+
+
+### X-Ray
+
+The Altis Dashboard now has X-Ray analytics functionality built in, providing application performance management (APM) features, empowering you to dig into your performance over time. This allows you to examine up to 24 hours of data over any period from the past 30 days, and break down performance informaton at an aggregate level.
+
+X-Ray now has [improved documentation](docs://cloud/dashboard/x-ray.md) to allow you to better use, understand, and build atop the X-Ray functionality.
+
+Additionally, data reported to X-Ray has been improved to remove noise and provide more accurate data. Internal healthcheck requests sent by Altis Cloud infrastructure are no longer reported in X-Ray, and database queries now use the final query sent to the database.
+
+
+## Other Features and APIs
+
+### Post Cloning
+
+The CMS module now includes the ability to clone posts, including their full post data and post metadata. This behaviour can be modified via the [provided actions and filters](docs://cms/post-cloner.md).
+
+### Post GUIDs
+
+In order to set the groundwork for future import/export and migration tooling, GUIDs for post objects in Altis now use UUID URNs (e.g. `urn:uuid:d72fdc38-1305-4568-9821-463120b250f3`). This replaces the former URL-style GUIDs used for posts (e.g. `http://altis.local/?p=42`).
+
+Due to legacy use of the GUID field, some older code may attempt to use the GUID for image media source URLs. This code must be updated to use `wp_get_attachment_url()` instead, which will also ensure that Tachyon and the Altis CDN are correctly used.
+
+New code should only use the GUID field to uniquely identify posts, and never as a URL.
+
+### Minimum version of the AWS SDK
+
+The minimum version of the AWS SDK included with Altis has been bumped to 3.150.0. We recommend keeping the SDK up-to-date via Dependabot.
```

Comment:
So we recommend this? Seems like a pain to call out specifically.

Reply:
Might drop this section entirely as it seems to be backwards compatible, but had included as an "FYI our minimum of this dep you might also be using is being bumped"